create.go

```diff
@@ -14,6 +14,9 @@ import (
 	"github.com/pkg/errors"
 )
 
+// CreateOptions contains options related to starting a process, including, but
+// not limited to things like its execution arguments, environment variables
+// and a timeout.
```

Comment:
I would delete "including.... " and replace with... this includes execution configuration, post-execution triggers, and output configuration.